inputfile/radii.txt

```diff
@@ -124,4 +124,4 @@ Number	Element	vdw radius
 115	Mc	0
 116	Lv	0
 117	Ts	0
-118	Og	0
\ No newline at end of file
+118	Og	0

```

Comment:
Not sure what changed.

Reply:
I think something about a end-of-line character.